backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/service/CardVerifier.kt

```diff
@@ -0,0 +1,30 @@
+package app.ehrenamtskarte.backend.verification.service
+
+import app.ehrenamtskarte.backend.verification.database.repos.CardRepository
+import app.ehrenamtskarte.backend.verification.domain.Card
+import com.eatthepath.otp.TimeBasedOneTimePasswordGenerator
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.time.Duration
+import java.time.Instant
+import javax.crypto.spec.SecretKeySpec
+
+val TIME_STEP: Duration = Duration.ofMinutes(5)
```

Comment:
I'd vote for 0.5 minutes aka 30 seconds

Reply:
Me too, but we should agree on that with the others

Reply:
I think, we can go for it ! :D